config/testnet/node_config.env

```diff
@@ -1,20 +1,20 @@
 #
 
-# Specify your public IP address or domain name with the open port
-# in NODE_ADDRESS and BIND_ADDRESS
+# Decide on the quantity and list your public IP address(es) or domain name(s) with the open port(s).
 # By default keep these addresses the same.
 
 # This is used for the public announcement in the network map
 #
-#NEOFS_NODE_ADDRESS=<ANNOUNCE_ADDRESS>
-#NEOFS_NODE_ADDRESS=1.1.1.1:36512
-NEOFS_NODE_ADDRESS=
+#NEOFS_NODE_ADDRESSES=1.1.1.1:36512 2.2.2.2:36513
+NEOFS_NODE_ADDRESSES=
 
-# This is used to open the connection listening socket on your machine
+# This is used to open the connection listening socket(s) on your machine
 #
-#NEOFS_GRPC_ENDPOINT=<BIND_ADDRESS>
-#NEOFS_GRPC_ENDPOINT=1.1.1.1:36512
-NEOFS_GRPC_ENDPOINT=
+#NEOFS_GRPC_NUM=2
+#NEOFS_GRPC_0_ENDPOINT=1.1.1.1:36512
+#NEOFS_GRPC_1_ENDPOINT=2.2.2.2:36513
+NEOFS_GRPC_NUM=
```

Comment:
For the testnet I suggest to keep this config as simple as possible and do not describe the possibility of group addresses at all (as we ignore TLS configuration there). The goal is to change config as less as possible for the user.

So I would do something like this.

```
#

# Specify your public IP address or domain name with the open port
# in NODE_ADDRESSES and GRPC_0_ENDPOINT
# By default keep these addresses the same.

# This is used for the public announcement in the network map
#
#NEOFS_NODE_ADDRESSES=1.1.1.1:36512
NEOFS_NODE_ADDRESSES=

# This is used to open the connection listening socket(s) on your machine
#
#NEOFS_GRPC_0_ENDPOINT=1.1.1.1:36512
NEOFS_GRPC_0_ENDPOINT=
NEOFS_GRPC_NUM=1


#https://unece.org/trade/cefact/unlocode-code-list-country-and-territory
# Find suitable LOCODE (from the link above) for your storage node.
#
#NEOFS_NODE_ATTRIBUTE_1=UN-LOCODE:<XX YYY>
#NEOFS_NODE_ATTRIBUTE_1=UN-LOCODE:RU LED
NEOFS_NODE_ATTRIBUTE_1=


NEOFS_NODE_ATTRIBUTE_2=Price:100000
```

Reply:
Changed